Task_1.py

```diff
@@ -0,0 +1,160 @@
+import os
+from source import get_data
+import csv
+from operator import itemgetter
+
+
+# 1) Task_1.py: to create file general-stats.cvs containing information about:
+#    1.1. average rating (overall) of each application (asin);
+def avg_rating(main_dict):
+    asins = []
+    for item in main_dict.values():
+        asins.append(item['asin'])
+    asins_value = dict.fromkeys(set(asins), [0, 0])
+    for item in main_dict.values():
+        asins_value[item['asin']] = [asins_value.get(item['asin'])[0] + item['overall'], asins_value.get(item['asin'])[1]+1]
+    avg_overall = []
+    for key, value in asins_value.items():
+        avg_overall.append([key, asins_value.get(key)[0] / asins_value.get(key)[1]])
+    return avg_overall
+
+# 1) Task_1.py: to create file general-stats.cvs containing information about:
+#    1.2. messages with the most “likes” from the entire data set and the application (asin) associated with it;
+def best_comment(main_dict):
+    likes = {}
+    for key, value in main_dict.items():
+        likes[key] = value['helpful'][0]
+    sort_likes = sorted(likes.items(), key=itemgetter(1), reverse=True)
```

Comment:
We can find best_comment in one run without creating an extra dictionary. You just need to use [max](https://docs.python.org/3/library/functions.html#max) function: `max(main_dict.values(), key=lambda review: review['helpful'][0])`